sdk/messaging/azservicebus/example_receiver_test.go

```diff
@@ -80,6 +93,18 @@ func ExampleReceiver_ReceiveMessages() {
 		err = receiver.CompleteMessage(context.TODO(), message, nil)
 
 		if err != nil {
+			var sbErr azservicebus.Error
```

Comment:
Looks like the error isn't being returned by reference.

Reply:
It is now and examples have been updated.